<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkingDirWidget</class>
 <widget class="QWidget" name="WorkingDirWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,0,0,0,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&amp;Working dir :</string>
     </property>
     <property name="buddy">
      <cstring>workingDirCB</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="EllipsisComboBox" name="workingDirCB">
     <property name="toolTip">
      <string>Selected working directory</string>
     </property>
     <property name="statusTip">
      <string>Selected working directory</string>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="workingDirPB">
     <property name="toolTip">
      <string>Open a directory selection dialog</string>
     </property>
     <property name="statusTip">
      <string>Open a directory selection dialog</string>
     </property>
     <property name="text">
      <string>&amp;Select</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="browseCB">
     <property name="toolTip">
      <string>Open a File Explorer window in the selected working dir</string>
     </property>
     <property name="statusTip">
      <string>Open a File Explorer window in the selected working dir</string>
     </property>
     <property name="text">
      <string>&amp;Open dir</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="logPB">
     <property name="text">
      <string>&amp;Log</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EllipsisComboBox</class>
   <extends>QComboBox</extends>
   <header>ellipsiscombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
